x-pack/plugin/ml/qa/native-multi-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/ml/integration/DatafeedJobsIT.java

```diff
@@ -146,6 +146,43 @@ public void testLookbackOnlyDataStream() throws Exception {
         waitUntilJobIsClosed(job.getId());
     }
 
+    public void testLookbackOneEmptyPattern() throws Exception {
+        client().admin().indices().prepareCreate("data-1")
+            .setMapping("time", "type=date")
+            .get();
+        long numDocs = randomIntBetween(32, 2048);
+        long now = System.currentTimeMillis();
+        long oneWeekAgo = now - 604800000;
+        long twoWeeksAgo = oneWeekAgo - 604800000;
+        indexDocs(logger, "data-1", numDocs, twoWeeksAgo, oneWeekAgo);
+
+        Job.Builder job = createScheduledJob("lookback-empty-pattern-job");
+        registerJob(job);
+        putJob(job);
+        openJob(job.getId());
+        assertBusy(() -> assertEquals(getJobStats(job.getId()).get(0).getState(), JobState.OPENED));
+
+        List<String> t = new ArrayList<>(2);
```

Comment:
`t`? 
Whats wrong with `u` or `f` or `q`

I think a unit test would be easier to understand and provide better coverage.